includes/reports/overview/list-table/class-sensei-reports-overview-list-table-abstract.php

```diff
@@ -139,14 +139,14 @@ public function generate_report() {
 
 		// Handle orderby.
 		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- No action, nonce is not required.
-		$orderby = sanitize_key( wp_unslash( $_GET['orderby'] ?? '' ) );
+		$orderby = $this->get_orderby_value();
 		if ( empty( $orderby ) || ! array_key_exists( esc_html( $orderby ), $this->get_sortable_columns() ) ) {
 			$orderby = '';
 		}
```

Comment:
I think removing this check might fix the sorting issues @merkushin is having.

Reply:
Jup, fixed here https://github.com/Automattic/sensei/pull/4973/commits/8e5f9a93ebb8de8bdf137228c45fbfe46f8867e1